src/lib/net/util/client.ts

```diff
@@ -0,0 +1,58 @@
+import { RunService } from "@rbxts/services";
+
+import { FunctionUtil } from "../../util/functions";
+import { Identifiers } from "./identifiers";
+import { ClientListener, Contents, Packet } from "./types";
+
+export namespace Client {
+	const queue: Array<Packet> = [];
+	const listeners: { [x: string]: Array<ClientListener> } = {};
+
+	/** @hidden */
+	export function init(): void {
+		const remote = script.Parent?.Parent?.WaitForChild("[...Tina]") as RemoteEvent;
```

Comment:
! not ? here if you expect it to exist.

Reply:
I **do expect it** to exists, but it may not.